core/iwasm/compilation/aot_emit_variable.c

```diff
@@ -83,7 +83,7 @@ aot_compile_op_set_or_tee_local(AOTCompContext *comp_ctx,
 
     if (comp_ctx->aot_frame) {
         /* Get the slot index */
-        n = comp_ctx->aot_frame->cur_wasm_func->local_offsets[local_idx];
```

Comment:
Could we also remove cur_wasm_func field in AOTCompFrame? Seems it is only used here.

Reply:
Yes, removed